src/pipecat/pipeline/service_switcher.py

```diff
@@ -142,3 +165,9 @@ async def process_frame(self, frame: Frame, direction: FrameDirection):
 
         if isinstance(frame, ServiceSwitcherFrame):
             self.strategy.handle_frame(frame, direction)
+            service_switcher_filter_frame = ServiceSwitcher.ServiceSwitcherFilterFrame(
+                active_service=self.strategy.active_service
+            )
+            # Hack: we need access ParallelPipeline internals here to queue the frame in each of the pipelines
+            for p in self._pipelines:
+                await p.queue_frame(service_switcher_filter_frame, direction)
```

Comment:
I haven't tried it yet, but this would be the equivalent of doing this, right:
`await super().process_frame(service_switcher_filter_frame, direction)`

Would it be better ? What do you think ?